README.md

```diff
@@ -1,2 +1,49 @@
 # form-gen-tool
-Dynamically create HTML forms
+
+*Dynamically create HTML forms*
+
+## Setup 
+
+### Dependencies
+
+* node
+* npm
+
+### Installation
+
+* Clone this repo:
+git@github.com:cayouetten/form-gen-tool.git
+
+## Test
```

Comment:
Consider renaming this to something like "Example" rather than "Test," which could have some specific QA connotations depending on who's reading this.